database-mongodb/data.js

```diff
@@ -30,7 +30,7 @@ const generateHosts = () => {
       "listingId": [shuffledIds.pop()],
       "fullName": faker.name.findName(),
       "joined": faker.date.past(),
-      "photoUrl": faker.image.people(),
+      "photoUrl": faker.image.avatar(),
       "superhost": faker.random.boolean(),
```

Comment:
what brought about this change?

Reply:
I realized the original photos were groups of people. I needed solo photos hence the name avatar.